packages/ui/views/app/index.tsx

```diff
@@ -6,9 +6,10 @@ import Engine, { EngineState } from 'lndr/engine'
 
 import AuthenticateView from 'ui/views/authenticate'
 import AccountView from 'ui/views/account'
+import WelcomeView from 'ui/views/welcome'
 
 import { PopupTarget } from 'ui/components/popup'
-import Logo from 'ui/components/logo'
+import Logo from 'ui/components/images/logo'
```

Comment:
Not ok to import something from the non-top-level of a component - components are just that - isolated pieces of functionality. This could be corrected by using the `ThemeImage` idea from earlier.